### PR TITLE
Add initial documentation for review

### PR DESCRIPTION
**Pull request by rchristman89**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,81 @@
...
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+### Features
 
-## Contributing
+- Azure Active Directory (AAD) Virtual Machine (VM) Join
+- On-Prem VM Join
+- Apply STIGs to deployed VMs
```
Comment (regedit32) [inline README.md:11]: Let's spell out STIG `DoD Security Technical Implementation Guides (STIG)`.  We could also mention `using PowerSTIG` and link to it in GitHub

File: README.md [hunk 2]
```diff
@@ -1,33 +1,81 @@
...
+- Simplified VM sizing based on Microsoft documentation
```
Comment (regedit32) [inline README.md:12]: Let's include the reference https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/virtual-machine-recs

File: README.md [hunk 3]
```diff
@@ -1,33 +1,81 @@
...
+
+When using a VHD from blob storage, during the deployment of the session host VMs a resource type Image is created and will show up in the resource group the AVD deployment deployed to.
+
+### Virtual Machine Generation
+
+When building an image in Hyper-V the VM you are building the image from has to be a Generation 1 VM. If not the deployment will fail and the VMs that are built will have a boot issue when looking at the boot diagnostics.
+
+### References
+
+These are references for building a VHD and ensuring it has the correct sepcifications plus important settings to set in the Windows Operating System.
```
Comment (regedit32) [inline README.md:77]: `specifications`

File: README.md (hunk 1 above)
Comment (rchristman89) [reply README.md]: Added and linked

File: README.md (hunk 3 above)
Comment (rchristman89) [reply README.md]: fixed

File: README.md [hunk 4]
```diff
@@ -1,33 +1,81 @@
...
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+## Getting Started
 
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+Depending on the if the VMs are going to be AAD or On-Prem joined there are differnt prerequisites that are needed.
```
[the remark this answers is not in the document]
Comment (rchristman89) [reply README.md]: fixed

**Pull request merged**